src/ResourceManagement/HDInsight/HDInsight.Tests/Helpers/GetClusterSpecHelpers.cs

```diff
@@ -32,6 +32,13 @@ public static class GetClusterSpecHelpers
         private const string HttpPassword = "";
         private const string RdpUser = "";
         private const string RdpPassword = "";
+        private const string VirtualNetworkId = "";
+        private const string SubnetName = "";
+        private const string DomainAdminUserName = "";
+        private const string DomainAdminPassword = "";
```

Comment:
this PR is missing changes for the customization stuff (i.e. ClusterCreateParameters)